src/library/scala/util/Properties.scala

```diff
@@ -57,7 +57,7 @@ private[scala] trait PropertiesTrait {
   def setProp(name: String, value: String)      = System.setProperty(name, value)
   def clearProp(name: String)                   = System.clearProperty(name)
 
-  def envOrElse(name: String, alt: String)      = Option(System getenv name) getOrElse alt
+  def envOrElse(name: String, alt: => String)   = Option(System getenv name) getOrElse alt
   def envOrNone(name: String)                   = Option(System getenv name)
 
   def envOrSome(name: String, alt: Option[String])       = envOrNone(name) orElse alt
```

Comment:
I think I suggested this before, but for completeness, convert all the API that correspond to Option, including `envOrSome`. And `scalaPropOrElse`?